### DISCUSSION TITLE
can we convert jpeb/png to webp and gif to webp using imageproxy?

### DISCUSSION THREAD

**Discussion by dhruv-nudge** [willnorris/imageproxy#392] [General]
I want to convert all image formats to webp
and all video/gif files to webm or mp4

can I do this using imageproxy?

**Comment by willnorris** [comment]
imageproxy does not currently support converting to webp, though that should be possible with some of the discussion in https://github.com/willnorris/imageproxy/issues/388.  imageproxy doesn't do any video conversion.
